.golangci.yml

```diff
@@ -25,6 +25,11 @@ linters:
     - golint
     - maligned
     - scopelint
+    # deprecated
```

Comment:
I would actually do the opposite, use `disable-all` and enable only the ones that make sense for the project...